src/chapters/html/semantic-html.rst

```diff
@@ -0,0 +1,305 @@
+   :lineno-start: 6
+
+   <body>
+      <header>Content...</header>
+      <section>
+         <article>
+            Content...
+            <figure>
+               <img src="..." alt="Text...">
+               <figcaption>Content...</figcaption>
+            </figure>
+         </article>
+      </section>
+      <footer>Content...</footer>
+   </body>
+
+Aha! Finding the figure caption is a snap now, and we can see that we have a
+header and footer at the top and bottom of the page. It looks like the purpose
+of the ``section`` element is to organize related ``article`` and ``figure``
+elements, and the ``article`` tag name gives us a clue about the type of
+content that begins on line 10.
+
+HTML Writing Style
+------------------
+
+.. index:: ! semantic html
+
+Just like any other author, programmers write HTML with different style guides
+and ways of thinking. Two different coders can look at the same set of content
+and choose different ways to arrange it on the screen. However, both of them
+use the same set of HTML tags to do their work.
+
+**Semantic HTML** is not about the appearance of the webpage. Instead, it's
+about the specific meaning of the elements. Semantic HTML uses more descriptive
+tag names to help identify what each section of the code does. This helps
+programmers clarify the structure of a webpage.
+
+For example, developers can make a paragraph larger than a heading if they want
+to. By looking at the HTML, however, we can still identify what is the
+paragraph and what is the heading. The ``<p>`` and ``<h_>`` tags give us that
+information. This is the strength of semantic HTML. The tag names express
+meaning.
+
+Another benefit to semantic HTML is that programmers find it easier to
+visualize the results when they can put a specific name to a section of their
+code.
+
+Screen Readers and Search Engines
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A well designed webpage organizes content in a clean and attractive way. For
+these pages, finding the navigation bar or identifying the main content of the
+page is easy. However, screen readers for visually impaired users verbally
+describe the content on a page. These devices cannot pick up on visual clues to
+identify what is the most important information.
+
+The ``div`` and ``span`` elements organize content on the page, but they do not
+provide any information about the content itself. If we use separate ``div``
+elements to build a navigation bar and organize a set of blog posts, a screen
+reader assigns equal importance to each element. If the nav bar is at the top
+of the page, the device will read the buttons and menu items first before
+moving to the posts. This is distracting and can provide a negative user
+experience.
+
+With semantic HTML tags, a screen reader can more easily identify the content
+on the page that should be read or emphasized. For example, content inside a
+``header`` element can be assigned less importance than content inside an
+``article``.
```

Comment:
This makes it sound like a screen reader skips over headers in favor of articles. Screen readers start with headers.

Reply:
Removed this paragraph.